interface/patient_file/summary/immunizations.php

```diff
@@ -465,7 +465,7 @@ function saveImmunizationObservationResults($id, $immunizationdata)
                     <div class="form-group mt-3">
                         <label>
                             <?php echo xlt('Date of VIS Statement'); ?>
-                            (<a href="https://www.cdc.gov/vaccines/pubs/vis/default.htm" title="<?php echo xla('Help'); ?>" rel="noopener" target="_blank">?</a>)
+                            (<a href="https://www.cdc.gov/vaccines/hcp/vis/current-vis.html" title="<?php echo xla('Help'); ?>" rel="noopener" target="_blank">?</a>)
```

Comment:
This change needs to be backed out as it was part of your old PR and isn't related to the vitals issue.

Reply:
Thank you for your response. I'll be backed out this change.

Reply:
This is still here and needs to be fixed.